### PR TITLE
Add accessibility documentation

### PR DESCRIPTION
**Pull request by ksiler**
This creates a page to document all things accessibility.

### CODE REVIEW

File: docs/pages/accessibility.mdx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+In the above example, Narrator will read the first button as "One, 1 of 2, level 1" and the second button as "Two, 2 of 
+2, level 1" and so on.
+
+### Roles
+```accessibilityRole``` communicates the purpose of the component to the user.
+
+### States
+
+```accessibilityStates``` describes the current state of a component to the user.
```
Comment (dotnetjunky) [inline docs/pages/accessibility.mdx:89]: Isn't React changing to `accessibilityState` prop?

File: docs/pages/accessibility.mdx [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# Accessibility
+
+Accessibility means designing and creating UI that's accessible to all users, including those with disabilities.
+It's important to create accessible UI so that assitive technology, like screen readers, can access information users 
+need to interact with your contorls. Here you can find a guide to get started as well as tools to help you test your UI
```
Comment (samuelfreiberg) [inline docs/pages/accessibility.mdx:5]: Spelling error: Should be "controls"

File: docs/pages/accessibility.mdx [hunk 3]
```diff
@@ -0,0 +1,104 @@
+# Accessibility
+
+Accessibility means designing and creating UI that's accessible to all users, including those with disabilities.
+It's important to create accessible UI so that assitive technology, like screen readers, can access information users 
+need to interact with your contorls. Here you can find a guide to get started as well as tools to help you test your UI 
+for accessibility.
+
+## Our Approach
+FluentUI React Native enables developers to create accessible UI by providing APIs that are complimentary to Microsoft
+UI Automation (UIA), an accessibility framework for Windows. Our API surface is heavly inspired by Accessible Internet Rich
```
Comment (samuelfreiberg) [inline docs/pages/accessibility.mdx:10]: "heavily" spelling

File: docs/pages/accessibility.mdx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+### Roles
+```accessibilityRole``` communicates the purpose of the component to the user.
+
+### States
+
+```accessibilityStates``` describes the current state of a component to the user.
+
+### Actions
+
+Accessibility actions allow assistive technology to programically invoke actions you provide to a component. In order to
```
Comment (samuelfreiberg) [inline docs/pages/accessibility.mdx:93]: "programmatically" spelling

File: docs/pages/accessibility.mdx (hunk 1 above)
Comment (ksiler) [reply docs/pages/accessibility.mdx:89]: Yes, but we currently only have support for accessibilityStates at the moment. When we add support for accessibilityState we should update the docs.

File: docs/pages/accessibility.mdx (hunk 2 above)
Comment (ksiler) [reply docs/pages/accessibility.mdx]: fixed in recent commit

File: docs/pages/accessibility.mdx (hunk 3 above)
Comment (ksiler) [reply docs/pages/accessibility.mdx]: fixed in recent commit

File: docs/pages/accessibility.mdx (hunk 4 above)
Comment (ksiler) [reply docs/pages/accessibility.mdx]: fixed in recent commit

**Comment by ksiler** [comment]
Closing because of branch issues - will reopen in new pull request.

**Pull request closed without merging**